src/Forms/Forms.Component.js

```diff
@@ -92,21 +92,18 @@ export const FormsComponent = () => {
 
             <Example
                 description={`The state of the input field can reflect validity of the data entered, 
-                    whether the input data is editable or disabled.\n\n* **Default**: The field is 
+                    whether the input data is editable or disabled.\n\n* **Normal**: The field is 
```

Comment:
This is still "default", isn't it?

Reply:
See comment below and https://sap.github.io/fundamental/components/form.html#input-states, although I'm totally on board with changing this, we should do it consistently.